deployments/server/values.yaml

```diff
@@ -20,3 +20,7 @@ enableServiceMonitor: false
 
 service:
   annotations:
+
+serviceAccount:
+  create: true
+  name:
```

Comment:
```suggestion
  serviceAccountName: inference-manager-engine
```

Reply:
wondering why we don't want `llm-operator-` prefix for this serviceaccount, and it is for `inference-manager-server` instead of `inference-manager-engine`?
```
kubectl get serviceaccounts -n llm-operator
NAME                                    SECRETS   AGE
default                                 0         14d
llm-operator-inference-manager-server   0         156m
llm-operator-job-manager-dispatcher     0         14d
llm-operator-job-manager-server         0         14d
```

Reply:
My bad. It should be `inference-managa-server`. Also it's better to just remove this line.

Then https://github.com/llm-operator/inference-manager/blob/main/deployments/engine/templates/_helpers.tpl#L59 sets the service account name.

As for the `llm-operator-` prefix, Helm adds it since the chart name (eg.., `job-manager-dispatcher`) is not as the release name we specify in `helm install`. 

https://stackoverflow.com/questions/66841290/helm-adds-release-name-to-dependent-charts

This is not particularly intentional and we can fix if we want.